src/server/plugins/engine/services/formsService.js

```diff
@@ -0,0 +1,39 @@
+import { formMetadataSchema } from '@defra/forms-model'
+
+import config from '~/src/server/config.js'
+import { getJson } from '~/src/server/plugins/engine/services/httpService.js'
+
+const { managerUrl } = config
+
+/**
+ * Retrieves a form definition from the form manager for a given slug
+ * @param {string} slug - the slug of the form
+ */
+export async function getFormMetadata(slug) {
```

Comment:
Initially I pulled in the `fetch.js` from `forms-designer`, with all the new `URL` goodness you added.
However, I reverted to using the pre-existing `import { getJson } from '~/src/server/plugins/engine/services/httpService.js'`. This is because there's a subtle differences in `Wreck.request(method, url)` vs `Wreck.get(url)`. These shorthand methods have [some useful error handling](https://github.com/hapijs/wreck/blob/master/lib/index.js#L547) built-in.

I then spent too long trying to get the types to play nicely with the shorthand methods until I eventually gave in and used what I already had at my disposal. I would be great to try and get it working together if we get a free hour (including maybe moving to native `fetch`).